src/realm/object-store/list.hpp

```diff
@@ -36,6 +36,11 @@ class ThreadSafeReference;
 struct ColKey;
 struct ObjKey;
 
+namespace object_store {
```

Comment:
Why this? Seems not to be needed

Reply:
This is forgotten code, I was trying to add also the new interface to object store types.